src/Core/src/Core/IEditor.cs

```diff
@@ -5,9 +5,14 @@
 	/// </summary>
 	public interface IEditor : IView, ITextInput, ITextStyle, ITextAlignment
 	{
+		/// <summary>
+		/// Gets a value that controls whether the editor will change size to accommodate input as the user enters it.
+		/// </summary>
+		EditorAutoSizeOption AutoSize { get; }
```

Comment:
We'll want to uncomment this [test](https://github.com/dotnet/maui/blob/b1bdc55fb7a6fa4457f67ec288f13077583acb93/src/Controls/tests/DeviceTests/Elements/Editor/EditorTests.cs#L16) as part of this PR

Ideally we don't have to move this property back into `Core` and we can just make it work inside `Controls` 

If we have to go the same route here with `FrameChanged` then we should put that code into controls.

Also, I think a different approach we can take here instead of having to use inheritance is to add an observer

```C#
uiView.Layer.AddObserver("bounds", Foundation.NSKeyValueObservingOptions.OldNew, (_) => OnLoadedCheck()
```

Reply:
Changed the PR to move the changes inside `Controls`. Could you review it again?. Thanks!.